fail2ban/tests/clientreadertestcase.py

```diff
@@ -649,8 +649,9 @@ def testReadStockJailConfForceEnabled(self):
 
 			# and we know even some of them by heart
 			for j in ['sshd', 'recidive']:
-				# by default we have 'auto' backend ATM
-				self.assertTrue(['add', j, 'auto'] in comm_commands)
+				# by default we have 'auto' backend ATM, but some distributions can overwrite it, 
+				# (e.g. fedora default is 'systemd') therefore let check it without backend...
+				self.assertTrue([True for i in comm_commands if len(i) == 3 and i[0:2] == ['add', j]])
```

Comment:
performance difference is imho negligible but readability is somewhat ... if you want the same -- make 2nd one a generator: `['add', j] in (c[:2] for c in comm_commands)` ;)